src/System.Private.Uri/src/Resources/Strings.resx

```diff
@@ -258,4 +258,13 @@
   <data name="IO_PathTooLong" xml:space="preserve">
     <value>The specified file name or path is too long, or a component of the specified path is too long.</value>
   </data>
+  <data name="Argument_ExtraNotValid" xml:space="preserve">
+    <value>Extra portion of URI not valid</value>
+  </data>
+  <data name="Argument_InvalidUriPart" xml:space="preserve">
+    <value>Uri part is not is not valid.</value>
```

Comment:
It's used when the system requests [just some part of the Uri](https://github.com/Clockwork-Muse/corefx/blob/nameof_fixup/src/System.Private.Uri/src/System/Uri.cs#L1269-L1310) (scheme, authority, path, query).  This is done via an Enum, which (since you can supply any int) throws if it's not one of the actual values, which is what the message is for.  We don't have an existing "not valid enum" type message, if that would have been better.